library/src/main/java/com/medium/highlightjs/models/Language.kt

```diff
@@ -0,0 +1,125 @@
+        "dns"
+    ),
+    DOS("dos"), BATCH("bat"), DART("dart"), DELPHI("delphi"), DIFF("diff"), DJANGO("django"), DOCKER_FILE("dockerfile"), DSCONFIG(
+        "dsconfig"
+    ),
+    DTS("dts"), DUST("dust"), DYLAN("dylan"), EBNF("ebnf"), ELIXIR("elixir"), ELM("elm"), ERLANG("erlang"), EXCEL("excel"), F_SHARP(
+        "fsharp"
+    ),
+    FIX("fix"), FLIX("flix"), FORTRAN("fortran"), G_CODE("gcode"), GAMS("gams"), GAUSS("gauss"), GDSCRIPT("godot"), GHERKIN("gherkin"), GN_FOR_NINJA(
+        "gn"
+    ),
+    GO("go"), GRAMMATICAL_FRAMEWORK("gf"), GOLO("golo"), GRADLE("gradle"), GROOVY("groovy"), HTML("html"), XML("xml"), HTTP("http"), HAML(
+        "haml"
+    ),
+    HANDLEBARS("hbs"), HASKELL("hs"), HAXE("hx"), HY("hy"), INI("ini"), INFORM7("i7"), IRPF90("irpf90"), JSON("json"), JAVA("java"), JAVA_SCRIPT(
+        "javascript"
+    ),
+    JOLIE("ol"), KOTLIN("kt"), LATEX("tex"), LASSO("lasso"), LEAF("leaf"), LEAN("lean"), LESS("less"), LDIF("ldif"), LISP("lisp"), LIVE_CODE_SERVER(
+        "livecodeserver"
+    ),
+    LIVE_SCRIPT("livescript"), LLVM("llvm"), LUA("lua"), MAKEFILE("makefile"), MARKDOWN("md"), MATHEMATICA("mma"), MATLAB("matlab"), MAXIMA(
+        "maxima"
+    ),
+    MAYA_EMBEDDED_LANGUAGE("mel"), MERCURY("mercury"), MIRC_SCRIPTING_LANGUAGE("mrc"), MIZAR("mizar"), MOJOLICIOUS("mojolicious"), MONKEY(
+        "monkey"
+    ),
+    MOONSCRIPT("moonscript"), N1QL("n1ql"), NSIS("nsis"), NGINX("nginx"), NIMROD("nimrod"), NIX("nix"), OBJECTIVE_CONSTRAINT_LANGUAGE(
+        "ocl"
+    ),
+    O_CAML("ocaml"), OBJECTIVE_C("objectivec"), OPENGL_SHADING_LANGUAGE("glsl"), OPEN_SCAD("scad"), ORACLE_RULES_LANGUAGE("ruleslanguage"), OXYGENE(
+        "oxygene"
+    ),
+    PF("pf"), PHP("php"), PARSER3("parser3"), PERL("perl"), PLAIN_TEXT("txt"), PONY("pony"), POSTGRE_SQL("pgsql"), POWER_SHELL("ps"), PROCESSING(
+        "processing"
+    ),
+    PROLOG("prolog"), PROPERTIES("properties"), PROTOCOL_BUFFERS("protobuf"), PUPPET("pp"), PYTHON("python"), PYTHON_PROFILER_RESULTS(
+        "profile"
+    ),
+    PYTHON_REPL("pycon"), Q("k"), QML("qml"), R("r"), RAZOR_CSHTML("razor"), REASON_ML("re"), RENDER_MAN_RIB("rib"), RENDER_MAN_RSL(
+        "rsl"
+    ),
+    ROBOCONF("roboconf"), RUBY("ruby"), RUST("rust"), SAS("sas"), SCSS("scss"), SQL("sql"), STEP_PART_21("p21"), SCALA("scala"), SCHEME(
+        "scheme"
+    ),
+    SCILAB("sci"), SMALI("smali"), SMALLTALK("smalltalk"), STAN("stan"), STATA("stata"), STYLUS("stylus"), SUB_UNIT("subunit"), SWIFT(
+        "swift"
+    ),
+    TEST_ANYTHING_PROTOCOL("tap"), TCL("tcl"), TEX("tex"), THRIFT("thrift"), TP("tp"), TWIG("twig"), TYPE_SCRIPT("typescript"), VB_NET(
+        "vbnet"
+    ),
+    VB_SCRIPT("vbscript"), VHDL("vhdl"), VALA("vala"), VERILOG("v"), VIM("vim"), X86_ASSEMBLY("x86asm"), XL("xl"), X_QUERY("xq"), ZEPHIR(
+        "zep"
+    );
+
+    companion object{
+        val MEDIUM_SUBLIST = listOf(
```

Comment:
Nitpick: This list should be in the `donkey` project as it is specific to the Medium app.